common/config/api.go

```diff
@@ -24,6 +29,12 @@ func APIFlags() []cli.Flag {
 			Name:   FLAG_INSTANCE,
 			EnvVar: ENVVAR_INSTANCE,
 		},
+		cli.DurationFlag{
+			Name:   FLAG_JOB_EXPIRY,
+			Value:  24 * time.Hour,
```

Comment:
this should be lower, maybe 1, 4, or 8 hours, and should use the `DEFAULT_JOB_EXPIRY` variable